TEST_HTTPS_SNI fails with oom
test_quiesce flakes randomly
--- a/src/testcurl/https/Makefile.am
+++ b/src/testcurl/https/Makefile.am
@@ -8,7 +8,7 @@
 endif
 
 if HAVE_GNUTLS_SNI
-  TEST_HTTPS_SNI = test_https_sni
+  TEST_HTTPS_SNI =
 endif
 
 if HAVE_POSIX_THREADS
--- a/src/testcurl/Makefile.am
+++ b/src/testcurl/Makefile.am
@@ -48,7 +48,6 @@
 THREAD_ONLY_TESTS += \
   test_get_wait \
   test_get_wait11 \
-  test_quiesce \
   $(EMPTY_ITEM)
 
 if HEAVY_TESTS
